### ISSUE TITLE
Stripping required spaces

### ISSUE THREAD

**Issue by studiosciences** [postcss/postcss-color-function#26]
```
#Input:
border-color: color(#111111 l(-5%)) color(#111111 l(-5%)) color(#111111 l(-15%));

#Output:
border-color: rgb(5, 5, 5)rgb(5, 5, 5)rgb(0, 0, 0);
```

I can repro this in the UI Next playground. I also see this with the webpack CSS Loader and minification disabled.

**Comment by TrySound** [comment]
Can be fixed with postcss-value-parser. @studiosciences Can you send pr?

**Comment by studiosciences** [comment]
I'll take a look, but I'm not so sure this will fall within my area of expertise.

**Comment by studiosciences** [comment]
@TrySound I wrote a couple of tests and the parser and stringifier didn't have issues. It detected the space between the functions as a node and retained it when stringified. So I'm back to looking at this plugin.

**Comment by studiosciences** [comment]
@TrySound ah, I for some reason thought this was an issue _with_ postcss-value-parser, rather than could be fixed by using it instead of balanced-match.

I'll try a quicker fix first. Using postcss-value-parser seems like a fix from someone else.

**Comment by TrySound** [comment]
@studiosciences Which exactly issue?

**Comment by MoOx** [comment]
Closed by #27
